compilers/concrete-optimizer/concrete-optimizer/src/dag/unparametrized.rs

```diff
@@ -194,7 +194,7 @@ impl<'dag> DagBuilder<'dag> {
         &mut self,
         inputs: impl Into<Vec<OperatorIndex>>,
         complexity: LevelledComplexity,
-        manp: f64,
+        weights: impl AsRef<[f64]>,
```

Comment:
is it better than "Into" ?
Except for speed maybe, but I assume array/Vec into() are optimized to move when possible.